### PR TITLE
fixing phoenix dex trades (merge after transfers PR)

### PR DESCRIPTION
**Pull request by andrewhong5297**
part of solana dex trades overhaul

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @andrewhong5297

File: models/phoenix/schema.yml
Comment (Hosuke) [inline models/phoenix/schema.yml:36]: New columns introduced for `phoenix_v1.trades`

**Comment by andrewhong5297** [comment]
This one has to wait until another pr is merged, just fyi!

**Comment by andrewhong5297** [comment]
@jeff-dude @aalan3 I'm using a seed in the main project, do I need to move that seed over? or I call it as a source now or something instead?

**Comment by andrewhong5297** [comment]
> @jeff-dude @aalan3 I'm using a seed in the main project, do I need to move that seed over? or I call it as a source now or something instead?

nvm got it

**Review by 0xRobin — APPROVED** [review]

File: solana/models/_sector/dex/phoenix/phoenix_v1_trades.sql
Comment (0xRobin) [reply solana/models/_sector/dex/phoenix/phoenix_v1_trades.sql:26]: ah, forgot we aren't materializing the seeds in prod. 
From the comment at L152:

> (l.tokenA_filled*coalesce(mm.raw_base_units_per_base_unit,1)) --base unit can be adjusted by phoenix, i.e. for BONK it starts at 1e6. There is a script for updating the markets seed file.

@andrewhong5297 can you convert this to a values model as we do for other static datasets?

**Pull request merged**
